mountpoint-s3/src/cli.rs

```diff
@@ -26,7 +26,7 @@ const AWS_CREDENTIALS_OPTIONS_HEADER: &str = "AWS credentials options";
 const LOGGING_OPTIONS_HEADER: &str = "Logging options";
 const CACHING_OPTIONS_HEADER: &str = "Caching options";
 const ADVANCED_OPTIONS_HEADER: &str = "Advanced options";
-
+const MOUNTPOINT_LOG_TARGET: &str = "mountpoint_s3";
```

Comment:
nit: let's move it away from the HEADER consts (either closer to the usage, or at least separated by a newline)